hack/install.sh

```diff
@@ -397,9 +397,10 @@ do_install() {
 			}
 
 			if [ "$lsb_dist" = "raspbian" ]; then
-				# Create Raspbian specific systemd drop-in file, use overlay by default
-				( set -x; $sh_c "mkdir -p /etc/systemd/system/docker.service.d" )
-				( set -x; $sh_c "echo '[Service]\nExecStart=\nExecStart=/usr/bin/dockerd --storage-driver overlay -H fd://' > /etc/systemd/system/docker.service.d/overlay.conf" )
+				# Create Raspbian specific systemd unit file, use overlay by default
+				( set -x; $sh_c "mkdir -p /etc/systemd/system" )
+				( set -x; $sh_c "$curl https://raw.githubusercontent.com/docker/docker/master/contrib/init/systemd/docker.service > /etc/systemd/system/docker.service" )
```

Comment:
From what I can see this script would install from either the main, test or experimental repo, leading to different docker versions, so I'm not sure if there's a way to get the commit hash at that point?

What about handling this at the debian package level? That would also avoid a dependency on github being up.